### ISSUE TITLE
Bug in txnbuild/change_trust.go func BuildXDR

### ISSUE THREAD

**Issue by adirutwn** [stellar/go#1296]
```
func (ct *ChangeTrust) BuildXDR() (xdr.Operation, error) {
	...
	op := xdr.Operation{Body: body}
	SetOpSourceAccount(&op, ct.SourceAccount)
	return xdr.Operation{Body: body}, nil
}
```

Should return op not the xdr.Operation{Body: body}

**Comment by ire-and-curses** [comment]
Thank you, great catch!

@poliha would you raise a PR for this to go in the 1.2 release?

    op := xdr.Operation{Body: body}
    SetOpSourceAccount(&op, ct.SourceAccount)
    return xdr.Operation{Body: body}, nil```

It should be

    op := xdr.Operation{Body: body}
    SetOpSourceAccount(&op, ct.SourceAccount)
    return op, nil```

**Comment by adirutwn** [comment]
just make a PR. Not sure if it works this way. But here's my PR -> https://github.com/stellar/go/pull/1298

**Comment by ire-and-curses** [comment]
Thanks for the PR. Normally we would accept that, but today we are in the middle of a release so needed to do this ourselves from the correct branch. Looking forward to your next contribution!

**Comment by ire-and-curses** [comment]
Fixed in #1299
